readme.rst

```diff
@@ -5,13 +5,15 @@ See the `Sphinx and Read the Docs <https://canonical-documentation-with-sphinx-a
 
 Then go through the following sections to use this starter pack to set up your docs repository.
 
+**Note:** After setting up your repository with the starter pack, you need to track the changes to the starter pack and manually update your repository with the required files.
+The `change log <https://github.com/canonical/sphinx-docs-starter-pack/wiki/Change-log>`_ lists the most relevant (and of course all breaking) changes, so keep an eye on it to make sure you don't miss important updates!
```

Comment:
i feel that we can leave this off, especially since this admonishment is already quite long:

", so keep an eye on it to make sure you don't miss important updates!"